tests/dummy/app/controllers/application.js

```diff
@@ -0,0 +1,10 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+
+export default class extends Controller {
+  @action
+  update(attribute, event) {
+    let value = event.target.value;
+    return this.model.set(attribute, value);
```

Comment:
It feels a bit iffy to modify `model`, but I understand that the PR is focused on updating the current syntax.

I might suggest that, in the future, we look at defining the validations and and the form "object" directly in the controller (currently these are defined in `tests/dummy/app/models/dummy.js`).

Reply:
Happy to do a review if you open a PR =)